tests/resources/functions/node-18.0/tests.js

```diff
@@ -47,7 +54,7 @@ module.exports = async (context) => {
             context.log(4.2);
             context.log(true);
 
-            return context.res.send();
+            return context.res.send('');
```

Comment:
What is the behaviour of `res.send()` and `res.send('')`?

Reply:
res.send should not be allowed and is not recommended. first argument is required.
res.send('') sends empty body, 200 status code, no headers.

I didnt notice this problem because JS allows anything. Python already complained that first argument is required

Reply:
okay got it 👍